properties/light.py

```diff
@@ -165,7 +165,7 @@ def update_is_laser(self, context):
     # sky2, sun, infinite, constantinfinite, area
     visibility_indirect_diffuse: BoolProperty(name="Diffuse", default=True, description=VIS_INDIRECT_DIFFUSE_DESC)
     visibility_indirect_glossy: BoolProperty(name="Glossy", default=True, description=VIS_INDIRECT_GLOSSY_DESC)
-    visibility_indirect_specular: BoolProperty(name="Specular", default=True, description=VIS_INDIRECT_SPECULAR_DESC)
+    visibility_indirect_specular: BoolProperty(name="Specular", default=False, description=VIS_INDIRECT_SPECULAR_DESC)
```

Comment:
This setting affects all lights specified in the comment above this section, not just the sun.

Reply:
How would you go about adding a condition if the light is a sun? Adding a new BoolProperty?

(I didn't know github would update my pull request with my latest change, I wanted these changes to be in another changelist for better task tracking, sorry about that)

Reply:
> (I didn't know github would update my pull request with my latest change, I wanted these changes to be in another changelist for better task tracking, sorry about that)

I thought so :)
Everything you push on this branch ends up in the pull request automatically.
So you have to revert this commit, open a new branch, then on the new branch cherry-pick this commit with `git cherry-pick -x 92322e07fb72694989c6878dc32bfbfb485c748c` (the hash of this commit).
https://stackoverflow.com/questions/19032296/how-to-use-git-revert
https://stackoverflow.com/questions/9339429/what-does-cherry-picking-a-commit-with-git-mean

> How would you go about adding a condition if the light is a sun? Adding a new BoolProperty?

I'll post a comment on the #52 issue.